api/experience/handlers.go

```diff
@@ -180,15 +180,15 @@ func UpdateUserExperienceByIdHandler(experienceSvc ExperienceService, c *gin.Con
 // @ID get-user-experience-details-by-id
 // @Accept  json
 // @Produce  json
-// @Param id path uint true "id"
```

Comment:
it will not work like this
you are passing `id` in path and fetching it from `experienceId`